contents/handbook/engineering/support-hero.md

```diff
@@ -32,6 +32,12 @@ The schedules consist of contiguous blocks, but that definitely doesn't mean wor
 
 Swap with a teammate in advance! Find a volunteer by asking in Slack, then use PagerDuty schedule overrides. You can trade whole weeks, but also just specific days. Remember not to alter the rotation's core order, as that's an easy way to accidentally shift the schedule for everyone.
 
+## I can't assign tickets or make public replies
+
+The HogHero app in the right sidebar should allow you to upgrade your user for your support week by clicking Full⬆️
```

Comment:
>  upgrade your user for your support week 

Can you add more information about what is expected here? I was unaware we are supposed to upgrade and downgrade. Some people are always checking tickets so it feels inefficient to save a few bucks.